### PR TITLE
Remove scroll-to-top button from print view

### LINKED ISSUE #1438: Omit 'scroll to top' button from the printable version
The 'scroll to top' button appears in the PDF file when converted to PDF. Can we omit it from _printable_ versions automatically?

image

### PR DESCRIPTION
**Pull request by wxwxwxwx9**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [x] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

This PR provides a quick fix that resolves #1438. 

**Overview of changes:**
Add a new global CSS rule for print media to hide any HTML elements with `no-print` class. 

**Anything you'd like to highlight / discuss:**
Whether the CSS rule should be placed elsewhere. 

**Testing instructions:**
`markbind serve` and ensure that the scroll-to-top button is in view on the browser. Open up the printing menu, you should notice that the scroll-to-top button is not visible in the print preview. It should not appear on the exported PDF file as well. 

**Proposed commit message: (wrap lines at 72 characters)**
Remove scroll-to-top button from print view

It is unlikely that users would want the scroll-to-top button to 
be visible in the printed version of the website. 

Let's enhance the scroll-to-top button feature by removing 
it from the printed version. 

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No blatantly unrelated changes
- [x] Pinged someone for a review!

### CODE REVIEW

**Comment by ang-zeyu** [comment]
would this work instead? 😮 

https://getbootstrap.com/docs/4.0/utilities/display/#display-in-print

**Comment by wxwxwxwx9** [comment]
> would this work instead? 😮
> 
> https://getbootstrap.com/docs/4.0/utilities/display/#display-in-print

This is so much more succint! Thanks for suggesting this. Let me switch the implementation to this.

**Review by ang-zeyu — APPROVED** [review]
Lgtm : )

Let's do an `updatetest` and we can merge this in

**Comment by ang-zeyu** [comment]
needs a rebase / merge with the latest as well

File: packages/cli/test/functional/test_site/expected/testCodeBlocks.html [hunk 1]
```diff
@@ -222,10 +222,10 @@ <h3 id="testing-site-nav"><span id="testing-site-nav" class="anchor"></span>Test
...
-</span><span>  <span class="hljs-tag">&lt;<span class="hljs-name">baz</span> <span class="hljs-attr">type</span>=<span class="hljs-string">&quot;name&quot;</span>&gt;</span>goo<span class="hljs-tag">&lt;/<span class="hljs-name">baz</span>&gt;</span>
-</span><span class="highlighted">  <span class="hljs-tag">&lt;<span class="hljs-name">qux</span> <span class="hljs-attr">type</span>=<span class="hljs-string">&quot;name&quot;</span>&gt;</span>goo<span class="hljs-tag">&lt;/<span class="hljs-name">qux</span>&gt;</span>
-</span><span class="highlighted">  <span class="hljs-tag">&lt;<span class="hljs-name">quux</span> <span class="hljs-attr">type</span>=<span class="hljs-string">&quot;name&quot;</span>&gt;</span>goo<span class="hljs-tag">&lt;/<span class="hljs-name">quux</span>&gt;</span>
+</span><span class="highlighted">  <span class="hljs-tag">&lt;<span class="hljs-name">bar</span> <span class="hljs-attr">type</span>=<span class="hljs-string">"name"</span>&gt;</span>goo<span class="hljs-tag">&lt;/<span class="hljs-name">bar</span>&gt;</span>
```
Comment (ang-zeyu) [inline packages/cli/test/functional/test_site/expected/testCodeBlocks.html:225]: 👀

Comment (wxwxwxwx9) [reply packages/cli/test/functional/test_site/expected/testCodeBlocks.html]: Thanks for guiding me on this :-) I have updated my outdated dependencies by running setup again.

**Pull request merged**